wp-content/plugins/civicrm/civicrm/ext/afform/core/ang/af/afForm.component.js

```diff
@@ -321,19 +321,24 @@
         var errorMessage = '';
         var isValid = true;
 
-        // Check if End is in the past
-        if (startDate < today) {
-            errorMessage += `Collection Camp Start Date ${startDateValue} cannot be in the past.\n`;
-            isValid = false;
+        // Check if the start date is in the past or today
+        if (startDate <= today) {
+          errorMessage += `Collection Camp Start Date ${startDateValue} cannot be today or in the past.\n`;
```

Comment:
Shouldn't the proper term be 'Collections Camp' rather than just 'Collections'? I think 'Collections Camp' conveys the message more clearly, what do you think ?

Reply:
This is Clear.